packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -21,7 +22,7 @@ export class OrgAuthInfo {
   ): Promise<string | undefined> {
     try {
       const defaultUserName = await ConfigUtil.getConfigValue(
-        DEFAULT_USERNAME_KEY
+        OrgConfigProperties.TARGET_ORG
```

Comment:
ConfigUtil.ts seems to be duplicated in three projects:
packages/salesforcedx-utils-vscode/src/config/configUtil.ts
packages/salesforcedx-utils-vscode/src/telemetry/configUtil.ts
packages/salesforcedx-vscode-core/src/util/configUtil.ts

If all three are the same, it would be good to consolidate and just have one copy.

Reply:
Also, there are two implementations of getDefaultUsernameOrAlias():
packages/salesforcedx-vscode-core/src/util/authInfo.ts
packages/salesforcedx-utils-vscode/src/auth/authUtil.ts

They differ, so I suggest not consolidating them, however, the other impl of getDefaultUsernameOrAlias()
has:
```
      const defaultUserName = await ConfigUtil.getConfigValue(
        DEFAULT_USERNAME_KEY
      );
```
...so my guess is that it too needs to be updated (?)

Reply:
I made an attempt at consolidation, but haven't been able to get the build working yet. Would you mind taking a look at https://github.com/forcedotcom/salesforcedx-vscode/pull/4296? Let me know if anything jumps out at you. Currently multiple extensions are crashing when I try to run the extensions after pointing them all to the same file and removing the duplicate files.